### ISSUE TITLE
Enhancement Request: Mobile Camera Integration and UI Bug on Mobile Devices

### ISSUE THREAD

**Issue by Bristopher** [manuel-rw/grocy-scanner#6]
First love the app it's so clean and exactly what I've been looking for!

I was wondering if there are any plans to integrate functionality that would allow using the website on an ios phone and being able to use the camera? Also if you plan on adding a mode/button to add straight to shopping cart.

Additionally, I've encountered a minor issue on mobile where clicking the fullscreen or minimize buttons in the bottom right corner results in an "error has occurred" message. 

Thank you!

**Comment by GoodOldJack12** [comment]
Most browsers block camera support if the site is not https, you might be running into that.

**Comment by Bristopher** [comment]
> Most browsers block camera support if the site is not https, you might be running into that.

That was it! It can see my iPhone's camera now, Thank you!

**Comment by manuel-rw** [comment]
> Also do you plan on adding a mode/button to add straight to a shopping list?

No, I do not have any plans for that since I don't use the shopping list feature. But I'm happy to review and accept any PRs.



Glad that you were able to solve the camera issue. Personally the camera is not stable enough for me and I purchased a barcode scanner which works great so far (instant read).

I'll leave this issue open so whoever has time can investigate the error (quite busy so let's see if I actually will have some capacity).

**Comment by GoodOldJack12** [comment]
Looks like its a compatibility issue with Safari: https://github.com/mdn/browser-compat-data/issues/18807
Simple css fix: https://stackoverflow.com/questions/71691474/css-media-query-display-mode-fullscreen-does-not-work-on-safari

As for shopping list feature, I was thinking of making a PR for that in the future.
